palisade-service/src/main/java/uk/gov/gchq/palisade/service/palisade/web/AuditClient.java

```diff
@@ -23,15 +23,25 @@
 
 import uk.gov.gchq.palisade.service.palisade.request.AuditRequest;
 
-import java.net.URI;
-
-@FeignClient(name = "audit-service", url = "undefined")
+/**
+ * The interface Audit client which uses Feign and uses services urls if provided, otherwise discovery by name with eureka.
+ */
+@FeignClient(name = "audit-service", url = "${web.client.audit-service}")
 public interface AuditClient {
-
+    /**
+     * Audit boolean.
+     *
+     * @param request the request
+     * @return the boolean
+     */
     @PostMapping(path = "/audit", consumes = "application/json", produces = "application/json")
-    Boolean audit(final URI url, @RequestBody final AuditRequest request);
+    Boolean audit(@RequestBody final AuditRequest request);
 
+    /**
+     * Gets health.
+     *
+     * @return the health
+     */
     @GetMapping(path = "/actuator/health", produces = "application/json")
-    Response getHealth(final URI url);
-
+    Response getHealth();
```

Comment:
all the other services and clients have removed the getHealth method, why not here?

Reply:
removed